From 327679a605129bda12df53345881a331c4449d46 Mon Sep 17 00:00:00 2001
From: Prasad Kumpatla <prasad.kumpatla@oss.qualcomm.com>
Date: Mon, 13 Jul 2026 23:39:06 +0530
Subject: [PATCH 07424/15333] ASoC: dt-bindings: qcom,sm8250: Add Hawi sound
 card

Add bindings for Hawi sound card, which is compatible with
the existing SM8450.

Signed-off-by: Prasad Kumpatla <prasad.kumpatla@oss.qualcomm.com>
Link: https://patch.msgid.link/20260713180907.874954-2-prasad.kumpatla@oss.qualcomm.com
Signed-off-by: Mark Brown <broonie@kernel.org>
---
 Documentation/devicetree/bindings/sound/qcom,sm8250.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/sound/qcom,sm8250.yaml b/Documentation/devicetree/bindings/sound/qcom,sm8250.yaml
index dae440ecab59..bd5f5a6268a3 100644
--- a/Documentation/devicetree/bindings/sound/qcom,sm8250.yaml
+++ b/Documentation/devicetree/bindings/sound/qcom,sm8250.yaml
@@ -24,6 +24,7 @@ properties:
       - items:
           - enum:
               - qcom,eliza-sndcard
+              - qcom,hawi-sndcard
               - qcom,kaanapali-sndcard
               - qcom,sm8550-sndcard
               - qcom,sm8650-sndcard
-- 
2.55.0

